### ISSUE TITLE
SENTENCES uncategorized

### ISSUE THREAD

**Issue by kaijagahm** [michaelchang64/ygdpDB#27]
```
s <- dbReadTable(con, "sentences")
table(s$sentenceType)

   CG check    CU    TS 
   14     5    10   194 
```

There are still 5 sentences that I didn't know how to categorize. Ask Jim.

**Comment by kaijagahm** [comment]
Checked master list--four of them (1025, 1025.1, 1026, 1026.1) are CG, one (1078) is CU.

Note that I personally don't find 1025 and 1025.1 to be particularly grammatical, or it's really a stretch. Probably would have marked those as like a 3 or a 2. Should reconsider whether they should be used as grammatical controls.

Wrote out text file, added info to meta_1.6.2. Ready for db update.

**Comment by kaijagahm** [comment]
Fixed on 1.6.2, 2020-09-23_2
